primitives/core/src/crypto.rs

```diff
@@ -354,275 +350,8 @@ fn ss58hash(data: &[u8]) -> blake2_rfc::blake2b::Blake2bResult {
 	context.finalize()
 }
 
-#[cfg(feature = "std")]
-lazy_static::lazy_static! {
-	static ref DEFAULT_VERSION: Mutex<Ss58AddressFormat>
-		= Mutex::new(Ss58AddressFormat::SubstrateAccount);
-}
```

Comment:
This is extreme Substrate specific, this should be in the ss58 crate.

The `ss58-registry` is a registry, nothing more. Aka only to host the enum

Reply:
Good point - I thought I couldn't `impl Default` due to trait orphan rules but as the enum is not being created in the ss58-registry crate I won't hit them. Will move default back out of the crate.